Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeInstance.java

```diff
@@ -361,16 +361,16 @@ public int hashCode() {
         @Override
         public String toString() {
             StringBuilder str = new StringBuilder(55);
-            str.append("(id=");
-            str.append(getId());
-            str.append(", displayName=");
-            str.append(getDisplayName());
-            str.append(", dbTableName=");
-            str.append(getDbTableName());
-            str.append(", supported=");
-            str.append(isSupported().toString());
-            str.append(", enabled=");
-            str.append(isEnabled().toString());
+            str.append("(id=")
+                    .append(getId());
+            str.append(", displayName=")
+                    .append(getDisplayName());
+            str.append(", dbTableName=")
+                    .append(getDbTableName());
+            str.append(", supported=")
+                    .append(isSupported().toString());
+            str.append(", enabled=")
+                    .append(isEnabled().toString());
```

Comment:
I think what codacy is complaining about is the reuse of the "str" variable. You should be able to chain all of these together in one statement (so you'll only use "str" once at the beginning).